### DISCUSSION TITLE
[Feature] Ability to hide face from the details screen

### DISCUSSION THREAD

**Discussion by jakob42** [immich-app/immich#4096] [Feature Request]
### The feature

Hi,

when viewing an image I sometimes see somebody is tagged in the bg I dont know. I would love to be able to easily hide that face since I now am sure I dont know them. I can get a detail view clicking on the face (where I can give it a name, merge it, set birthday), but I cannot hide it from there. But finding the face-thumb in the long list of persons is nearly impossible and not worth it.

Thanks for considering!

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
This is being worked on, so converting it to an issue for tracking.
